README.md

```diff
@@ -1,2 +1,34 @@
 # song-qc-sidecar
 A QC sidecar for Song that performs icgc-argo specific validations of song submissions
+
+**Creating a verifier program**
+
+There is a sample verifier program (written in Python) named *verifier.py* -- just edit it to create whatever verifier program you like.
+
+_verifier.py_ must define a single function, called **verify()**. The **verify()** function must take a string containing JSON as input, and must return a list of issues (strings describing problems with the input). If there are no issues, **verify()** must return an empty list.
+
+**Testing:**
+
+Files which contain sample inputs for which the verifier program should fail (produce issues) should be placed into the directory tests/failing. Files which contain sample inputs for which it should produce no issues should be placed into the directory tests/passing.
+
+To run the tests, run 
+
+```
+python3 -m pytest
```

Comment:
although this runs the test, having a bit verbosity may help the developer ensure their fixture ran. change this to
`python3 -m pytest -s -v`. 

The output
```
Testing file no_bai_file.json (should fail)
FAIL (good -- failure expected)
```
is super useful!